Documentation/components/drivers/special/ioexpander.rst

```diff
@@ -1,3 +1,35 @@
 ==========================
 IO Expander Device Drivers
 ==========================
+
+-  ``include/nuttx/ioexpander/ioexpander.h`` and ``include/nuttx/ioexpander/gpio.h``.
+   All structures and APIs needed to work with ioexpander drivers are provided in
+   this header file.
+
+-  ``struct ioexpander_ops_s``. Each ioexpand device driver must implement
+   an instance of ``struct ioexpander_ops_s``. That structure defines a
+   call table with the methods, and we also provide macros to help access methods.
+
+-  we also provide method ``gpio_lower_half`` to make ioexpander compatible with normal gpio.
+
+-  **Binding ioexpander Drivers**. ioexpander drivers are not normally directly
+   accessed by user code, we should always get lower level drivers, for example I2C,
+   and map extended gpio feature same asa normal gpio. See for example,
```

Comment:
```suggestion
   and map extended gpio feature same as a normal gpio. See for example,
```

Reply:
this patch come from master branch, let's fix it there and cherry-pick to release branch.